### PR TITLE
feat: Adding Parquet Support

### PR DESCRIPTION
**Pull request by mattwise-42**
Adding support for reading and writing parquet files.
Added additional output type command line parameter `-ot`.
Minor code improvements.
Updated documentation to reflect changes.

### CODE REVIEW

File: src/main/java/com/truveta/opentoken/CommandLineArguments.java [hunk 1]
```diff
@@ -11,14 +11,17 @@
...
     @Getter
     @Parameter(names = { "-h",
-            "--hashingsecret" }, description = "Get Hashing Secret for the tokengen workflow instance.", required = false)
+            "--hashingsecret" }, description = "Hashing Secret to hash token signatures.", required = false)
     private String hashingSecret = null;
 
     @Getter
     @Parameter(names = { "-e",
-            "--encryptionkey" }, description = "Get encryption key for tokengen workflow instance.", required = false)
+            "--encryptionkey" }, description = "Encryption key to encrypt tokens with.", required = false)
```
Comment (theworknish) [inline src/main/java/com/truveta/opentoken/CommandLineArguments.java:24]: The change from  `Get Hashing Secret for tokengen workflow instance` to `Get encryption key for tokengen `workflow instance. - makes much more sense!

File: .vscode/launch.json [hunk 2]
```diff
@@ -6,11 +6,11 @@
     "configurations": [
         {
             "type": "java",
-            "name": "OpenToken",
+            "name": "open-token",
             "request": "launch",
             "mainClass": "com.truveta.opentoken.Main",
             "projectName": "open-token",
-            "args": "-i src/main/resources/sample.csv -t csv -o target/output.csv -h \"HashingKey\" -e \"Secret-Encryption-Key-Goes-Here.\""
+            "args": "-i src/test/resources/sample.csv -t csv -o target/output.csv -ot csv -h \"HashingKey\" -e \"Secret-Encryption-Key-Goes-Here.\""
```
Comment (theworknish) [inline .vscode/launch.json:13]: So we are supporting csv and parquet now, right?

File: src/main/java/com/truveta/opentoken/Main.java [hunk 3]
```diff
@@ -30,15 +32,22 @@ public static void main(String[] args) throws IOException {
...
+            logger.info("Encryption Key: {}", maskString(encryptionKey));
+        }
         logger.info("Input Path: {}", inputPath);
         logger.info("Input Type: {}", inputType);
         logger.info("Output Path: {}", outputPath);
+        logger.info("Output Type: {}", outputType);
 
-        if (!"csv".equals(inputType)) {
-            logger.error("No input type other than csv supported yet!");
+        if (!(CommandLineArguments.TYPE_CSV.equals(inputType) || CommandLineArguments.TYPE_PARQUET.equals(inputType))) {
```
Comment (theworknish) [inline src/main/java/com/truveta/opentoken/Main.java:49]: do we need to test for null/empty `inputType`?

**Review by theworknish — APPROVED** [review]

**Review by theworknish — APPROVED** [review]
